app/uninstall.go

```diff
@@ -79,9 +79,10 @@ func uninstall(c *cli.Context) error {
 	}
 
 	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, time.Second*30)
+	kubeFilteredInformerFactory := informers.NewFilteredSharedInformerFactory(kubeClient, time.Second*30, namespace, nil)
 	lhInformerFactory := lhinformers.NewSharedInformerFactory(lhClient, time.Second*30)
 
-	ds := datastore.NewDataStore(lhInformerFactory, lhClient, kubeInformerFactory, kubeClient, extensionsClient, namespace)
+	ds := datastore.NewDataStore(lhInformerFactory, lhClient, kubeInformerFactory, kubeFilteredInformerFactory, kubeClient, extensionsClient, namespace)
 
```

Comment:
Same as above:
* looks like NewFilteredSharedInformerFactory is deprecated
* Looks like we forgot to start kubeInformerFactory

Reply:
Updated. Thank you.

Reply:
Good catch @PhanLe1010